src/ReleaseHistory.md

```diff
@@ -1,5 +1,9 @@
 # BinSkim Release History
 
+## Unreleased
+
+* FEATURE: support analyzing binaries build with Clang when command line was recorded. [#548](https://github.com/microsoft/binskim/pull/548)
```

Comment:
>* FEATURE: support analyzing binaries build with Clang when command line was recorded. [](http://example.com/codeflow?start=0&length=86)

It looks like there's also a change to which sarif-sdk is referenced. Can we get documented here and in the description the nature of that update?

Reply:
great the dependabot lag updating the sdk sub module,
I just see it got updated in main today,
so I have merged from latest main, this change is gone now.